### ISSUE TITLE
[SmartyPants] is incompatible with UsePipeTables

### ISSUE THREAD

**Issue by madskristensen** [lunet-io/markdig#13]
Using SmartyPants like this stops PipeTables from working:

```c#
var builder = new MarkdownPipelineBuilder()
    .UseSmartyPants()
    .UseAdvancedExtensions();

```

**Comment by xoofx** [comment]
This is a known issue. SmartyPants is breaking some expected layout in the syntax tree and things like Pipetables are not working well. That's why it is not part of the "Advanced" extensions.

It may require a bit more work to get this working, not sure I will fix this right away.

**Comment by madskristensen** [comment]
No worries. I've disabled SmartyPants for now

**Comment by xoofx** [comment]
Should be fixed by commit cff2b9a
